examples/asr/conf/fastconformer/fast-conformer-long_ctc_bpe.yaml

```diff
@@ -0,0 +1,203 @@
+# It contains the default values for training a Fast Conformer-CTC ASR model, large size (~120M) with CTC loss and sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+
+# You may find more info about Fast Conformer here: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#fast-conformer
+
+# This version uses Longformer-style attention in order to handle longer audio
+
+name: "FastConformer-CTC-BPE"
+
+model:
```

Comment:
@titu1994 does hydra have a feature that we create a config based on another config to avoid duplication of the conformer config every time? Just put the differences?

Reply:
Yeah it's one of its main features. However, NeMo is supposed to have only flat configs 1:1 corresponding to some script. Such hierarchical configs do reduce duplication yes, but also complicate running code cause now you'll need to have full directories of configs just to be able to link everything together. 

All Nemo configs should be fully self contained